drivers/hmis/app/models/hmis/role.rb

```diff
@@ -93,10 +93,10 @@ def self.permissions_with_descriptions
           'Administration',
         ],
       },
-      can_delete_assigned_project_data: {
-        description: 'Grants access to delete project related data for projects the user can see',
```

Comment:
I think this was like added as an example when HMIS was set up. We've never used it. We have separate perms for deleting different types of records.